app/views/pages/home.html.erb

```diff
@@ -1,3 +1,13 @@
+
+
+<%= render "shared/banner"%>
+
+<h1 data-controller="hello">Pages#home</h1>
+<p>Find me in app/views/pages/home.html.erb</p>
+<%= render "pages/partials/membership" %>
+
+
+
```

Comment:
This is a duplicate of the code at line 11 to 20. Please remove it

Reply:
I delleted line 8